assets/airlines/lpa.json

```diff
@@ -0,0 +1,17 @@
+{
+    "icao": "lpa",
+    "name": "Air Leap",
+    "callsign": {
+        "name": "Leap",
+        "callsignFormats": [
+            "####"
+        ]
+    },
+    "fleets": {
+        "default": [
+            ["SB20", 8],
```

Comment:
They only have one SB20

Reply:
According to my original source they had 3 Saab 2000, I have no idea why it says eight there. Guess I was a bit tired and read it wrong, they kind of look the same...

However, I see that your source says that they have 1 active and 2 retired, so I guess my source was just a bit old (it said august 2020, but I guess the pandemic forced them to retire 2 since then). I've changed to 1.